src/app/zap-templates/zcl/data-model/chip/client-monitoring-cluster.xml

```diff
@@ -18,40 +18,54 @@ limitations under the License.
+    <struct name = "MonitoringRegistrationStruct" isFabricScoped="true">
         <cluster code="0x1046"/>
         <item fieldId="1" name="ClientNodeId" type="node_id"/>
-        <item fieldId="2" name="ICid" type="INT64U"/>
+        <item fieldId="2" name="Key" type="OCTET_STRING" length="16"/>
     </struct>
 
     <cluster>
         <domain>General</domain>
         <name>Client Monitoring</name>
         <code>0x1046</code>
         <define>CLIENT_MONITORING_CLUSTER</define>
-        <description>Client Monitoring allows for ensuring that listed clients meet the required monitoring conditions on the server.</description>
-        <attribute side="server" code="0x00" define="IDLE_MODE_INTERVAL" type="INT32U" min="0x12C" max="0X5265C00" default="0x12C" writable="false" optional="false" isNullable="false">IdleModeInterval</attribute>
-        <attribute side="server" code="0x01" define="ACTIVE_MODE_INTERVAL" type="INT32U" min="0x12C" max="0X5265C00" default="0x12C" writable="false" optional="false" isNullable="false">ActiveModeInterval</attribute>
-        <attribute side="server" code="0x02" define="ACTIVE_MODE_THRESHOLD" type="INT16U" min="0x12C" max="0xEA60" default="0xFA0" writable="false" optional="false" isNullable="false">ActiveModeThreshold</attribute>
+        <description>Client Monitoring allows for ensuring that listed clients are notified when a server is available for communication.</description>
+        <attribute side="server" code="0x00" define="IDLE_MODE_INTERVAL" type="INT32U" min="500" max="86400000" default="500" writable="false" optional="false" isNullable="false">IdleModeInterval</attribute>
+        <attribute side="server" code="0x01" define="ACTIVE_MODE_INTERVAL" type="INT32U" min="300" default="300" writable="false" optional="false" isNullable="false">ActiveModeInterval</attribute>
+        <attribute side="server" code="0x02" define="ACTIVE_MODE_THRESHOLD" type="INT16U" min="300" default="300" writable="false" optional="false" isNullable="false">ActiveModeThreshold</attribute>
 
-        <!-- TODO: Revisit the number of entries per fabric-->
-        <attribute side="server" code="0x03" define="EXPECTED_CLIENTS" type="ARRAY" entryType="MonitoringRegistration" length="4" writable="false" optional="false" isNullable="false">ExpectedClients</attribute>
+        <attribute side="server" code="0x03" define="EXPECTED_CLIENTS" type="ARRAY" entryType="MonitoringRegistrationStruct" writable="false" optional="false" isNullable="false" reportable="false">
+            <description>ExpectedClients</description>
+            <access op="read" privilege="administer"/>
+            <access modifier="fabric-scoped"/>
+        </attribute>
 
-        <command source="client" code="0x00" name="RegisterClientMonitoring" optional="false">
+        <attribute side="server" code="0x04" define="ICD_COUNTER" type="INT32U" writable="false" optional="false" isNullable="false" reportable="false">
```

Comment:
```suggestion
        <attribute side="server" code="0x04" define="ICD_COUNTER" type="INT32U" writable="false" optional="false" isNullable="false">
```
Again, "reportable" is not related to Matter...  we do need a way to represent "C" in XML, but we don't have one right now.